infra/modules/bootstrap/main.tf

```diff
@@ -8,7 +8,7 @@ data "aws_partition" "current" {}
 # control over the key. This allows for ability to restrict access by key as well as policies attached to roles or users. 
 # https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html
 resource "aws_kms_key" "terraform_lock" {
-  description             = "KMS key for DynamoDB"
+  description             = "KMS key for DynamoDB table ${var.dynamodb_table}"
```

Comment:
unrelated, but would be nice if this variable was named a little more explicitly 😄  `dynamodb_table_name` or even `tfstate_locks_table_name`

(I probably reviewed the original PR so 🤷)

Reply:
Yeah I had the same thought! But then after poking around I wondered if it was intentional since it's actually called `dynamodb_table` in the terraform s3 backend (https://www.terraform.io/language/settings/backends/s3#dynamodb_table). What do you think? If you think we should rename it I'm happy to rename it in a separate PR.

Reply:
If we both had the same thought, we should probably rename it to something more user-friendly, even if it maps to `dynamodb_table` within the code. 🤷

Reply:
sounds good created #121 so i don't forget